src/world/common/UnkFunc46.inc.c

```diff
@@ -0,0 +1,20 @@
+#include "common.h"
+#include "map.h"
+
+ApiStatus N(UnkFunc46)(ScriptInstance* script, s32 isInitialCall) {
```

Comment:
Did you decide to not do the `static void* ptr = NULL;` thing? That's fine, just worth tracking it for #280

Reply:
We didn't do it for now, since the data has to be fully migrated to do it - And doing it half-half for those already migrated, and those that haven't been is kinda yucky.